apps/wallet/src/shared/sentry.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as Sentry from '@sentry/react';
+import { BrowserTracing } from '@sentry/tracing';
+
+const SENTRY_DSN =
+    'https://e52a4e5c90224fe0800cc96aa2570581@o1314142.ingest.sentry.io/6761112';
+
+const IS_PROD = process.env.NODE_ENV === 'production';
+
+export default function initSentry() {
+    if (!IS_PROD) return;
+
+    Sentry.init({
```

Comment:
we should add the [release](https://docs.sentry.io/platforms/javascript/configuration/options/#release) as well I think. Either the commit id or maybe the version from the package.json.

Reply:
This is done.